handlers/create_lease.go

```diff
@@ -30,6 +31,16 @@ func CreateLease(
 			return
 		}
 
+		authToken := os.Getenv("AUTH_TOKEN")
+		if authToken == "" {
+			htp.Error(w, http.StatusInternalServerError, "Server was not started with an auth token.")
+			return
+		}
```

Comment:
instead of the above, let's put the above code into our [config struct](https://github.com/deis/k8s-claimer/blob/master/config/server.go), and pass the auth token into `CreateLease`